Firmware/mmu2_protocol_logic.cpp

```diff
@@ -316,6 +316,7 @@ StepStatus ProtocolLogic::CommandStep() {
         SendFINDAQuery();
         return Processing;
     case ScopeState::FINDAReqSent:
+        findaPressed = rsp.paramValue;
         SendReadRegister(4, ScopeState::StatisticsSent);
         scopeState = ScopeState::StatisticsSent;
```

Comment:
even this line can be removed, I think SendReadRegister does that already

Reply:
Done, this saved 2 bytes of flash. See 279aa8e